api/python/quilt3/packages.py

```diff
@@ -305,6 +306,15 @@ def physical_keys(self):
         return [self.physical_key]
 
 
+class PackageUpdatePolicy(enum.Enum):
+    INCOMING = 'incoming'
+    EXISTING = 'existing'
+
+    @classmethod
+    def has_value(cls, value):
+        return value in cls._value2member_map_
```

Comment:
I'd rather [omit values](https://docs.python.org/3/library/enum.html#omitting-values) and in `set_dir()` check that `type(update_policy) is PackageUpdatePolicy`.

Reply:
checking `type(update_policy)` would require `update_policy` argument to be strictly an enum of type `PackageUpdatePolicy` which implies that passing string `incoming` or `existing` would fail.

Reply:
I think this behavior is desirable.
@akarve what do you think?

Reply:
I'm emulating pandas kwargs like parser=STRING. Easier for the user to just pass a string. I initially told Greg to use an enum but it's looking clumsy. I think we just do:
```
UPDATE_POLICY = set(
    'incoming',
    'existing',
)
```

And check `if update_policy in UPDATE_POLICIES`.

Reply:
Alright, I will update now.